rfd/0036/projects/triton-cli.md

```diff
@@ -0,0 +1,75 @@
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+
+<!--
+    Copyright 2016 Casey Bisson, Joyent
+-->
+
+# Managing projects with the Triton CLI
+
+## `triton (projects|project list|project ls)`
+
+List all projects in an organization.
+
+## `triton project (add|create|new) <project name> <project manifest>`
+
+Add a project to the organization. The [project manifest is defined elsewhere](manifest.md).
+
+## `triton project (get|show) <project uuid or name>`
+
+Show the project manifest and details for the specified project.
+
+## `triton project update <project uuid or name> <project manifest> (rolling=<positive int>) (canary|count=<positive int>)`
+
+Add a new project version with the given manifest, set that project version as the default, and trigger a `reprovision` of all service instances to the new version.
+
+Optional arguments passed through to [`reprovision`](#triton-service-reprovisionrestart-service-uuid-or-name-versionversion-uuid-imageimagespectag-instancenameuuid-compute_nodeuuid-countcanarypositive-integer-rollingpositive-integer) (see defaults in `reprovision`):
```

Comment:
This made sense in the context of services, but it makes much less sense in the context of a whole project. It's doubtful that the same rules should apply to every service of the project.